src/tokens.scss

```diff
@@ -0,0 +1,6 @@
+:host {
+  --ds-auro-popover-arrow-boxshadow-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
+  --ds-auro-popover-boxshadow-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
```

Comment:
These should *always* be the same color. We can use line 3 token anywhere you are using line 2.